setup.py

```diff
@@ -17,7 +17,7 @@
     'psutil',
     'six',
     'future',
-    'schema',
```

Comment:
Schema 0.7.0 seems to introduce some undesired behavior. For example, interactive_handler option has a default value RunnableHandler class. Schema 0.7.0 considered it to be a callable and called it with (). This actually evoked its __init__ rather than __call__ but __init__ requires some positional parameter.

            ConfigOption('interactive_handler', default=RunnableIHandler): object,

            for default in defaults:
                new[default.key] = default.default() if callable(default.default) else default.default

For now it will be good for us to stay on 0.6.6, we can file an issue for schema project.

Reply:
As discussed, for this sort of incompatibility it's preferable to specify a range/limit (<0.7.0) rather than pinning to an exact version. Since we are a library, we shouldn't pin versions too much.